src/NRules/NRules.Testing/CachedRuleActivator.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NRules.Fluent;
+using NRules.Fluent.Dsl;
+
+namespace NRules.Testing;
+
+public class CachedRuleActivator : IRuleActivator
```

Comment:
@gritcsenko we'll need XML docs on public types, but we can do it separately, after we settle on the code change (can even be a separate PR)

Reply:
Added XML Docs to all public types and members